frontend/src/js/components/ShoppingCart.jsx

```diff
@@ -53,8 +53,9 @@ var ShoppingCart = React.createClass({
           {tickets.map(function(ticket) {
             return (
               <Ticket
-                key={ticket.get('seat').id}
+                key={ticket.get('seat_id')}
                 ticket={ticket}
+                conflicts={this.props.conflictingSeatIds.indexOf(ticket.get('seat_id')) >= 0}
```

Comment:
a style question, I'll let you to decide, but I kind of dislike .indeOf() >= 0 and prefer `_.contains()` or a custom helper function